pkg/landscaper/registry/components/manager.go

```diff
@@ -75,16 +71,3 @@ func (m *Manager) ResolveWithBlobResolver(ctx context.Context, repoCtx cdv2.Repo
 	}
 	return client.ResolveWithBlobResolver(ctx, repoCtx, name, version)
 }
-
-// SetupManagerFromConfig returns a new Manager instance initialized with the given OCI configuration
-func SetupManagerFromConfig(log logging.Logger, config *config.OCIConfiguration, cacheIdentifier string) (*Manager, error) {
```

Comment:
Is there any reason why this function has been completely removed? Was it unused?

Reply:
yes it was not used